.evergreen/run-custom-csfle-tests.sh

```diff
@@ -104,7 +106,8 @@ if [ $DRIVER_CSFLE_TEST_RESULT -ne 0 ]; then
   exit 1
 fi
 
-echo "Test legacy version of FLE bindings"
-rm -rf node_modules/mongodb-client-encryption
-npm install mongodb-client-encryption@"^1.2.7"
-npm run check:csfle
+# These will now obviously fail as KMIP doesn't exist in the old bindings.
```

Comment:
I commented this out but I think we can just removed, since the new tests will no longer run against the legacy module as KMIP support is not backported.